packages/syftcli/src/bundle/create.py

```diff
@@ -0,0 +1,92 @@
+# future
+from __future__ import annotations
+
+# stdlib
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+# third party
+from rich import print
+from typer import Exit
+from typer import Option
+from typing_extensions import Annotated
+
+# relative
+from ..core.container_engine import ContainerEngine
+from ..core.container_engine import Docker
+from ..core.container_engine import Podman
+
+LATEST_VERSION = "0.8.2-beta.6"
+DEFAULT_TAR_FILE = Path("./syft_images/package.tar")
```

Comment:
MAYBE: We generally tend to  use folders like 

~/.syft
~/.hagrid

for storing files related to syft or hagrid

We could store the images under ~/.syft instead of current directory

Reply:
Updated. Accepts directory, instead of file path. Generates package with the following name `syft-{version}-{engine}.tar`